src/js/components/Select/Select.js

```diff
@@ -15,7 +15,7 @@ class Select extends Component {
     delete rest.children;
     const content = React.isValidElement(value) ? value : (
       <TextInput
-        margin='none'
+        theme={{ textInput: { extend: 'cursor: pointer' } }}
```

Comment:
Why via `theme` instead of just `style`?

Reply:
I wanted to use `TextInput.extend` but the HOCs would not allow me to do it. I'm trying to find a solution to use something like this:

```
const SelectInput = TextInput.extend`
  cursor: pointer;
`;
```

I like this solution because it will give us more flexibility if we need to add more styles to it in the future.

Reply:
But I guess to support something like this `TextInput` we would need to allow overriding `StyledTextInput` which can add more complexity. I will use style :)